<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabReserveRegister">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>1061</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Acesso</string>
     </attribute>
     <widget class="QCommandLinkButton" name="identifyUserButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>372</width>
        <height>53</height>
       </rect>
      </property>
      <property name="text">
       <string>Identificar Usuário</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="takeMorePictures">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>381</width>
        <height>53</height>
       </rect>
      </property>
      <property name="text">
       <string>Tirar Mais Fotos do Usuário</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cadastrar Usuário</string>
     </attribute>
     <widget class="QWidget" name="personForm" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>801</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
alternate-background-color: rgb(243, 243, 243);</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="firstNameLabel">
         <property name="text">
          <string>Nome:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="firstNameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lastNameLabel">
         <property name="text">
          <string>Sobrenome:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lastNameEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="cpfLabel">
         <property name="text">
          <string>CPF:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="cpfLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="userTypeLabel">
         <property name="text">
          <string>Tipo de Usuário:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="userTypeComboBox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Professor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Estudante</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Funcionário</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="registryLabel">
         <property name="text">
          <string>Matrícula:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="registryLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="functionalRegistryLabel">
         <property name="text">
          <string>Registro Funcional:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="functionalRegistryLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="employeeTypeLabel">
         <property name="text">
          <string>Tipo de Funcionário:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="employeeTypeLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="companyLabel">
         <property name="text">
          <string>Empresa:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="companyComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CPD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Empresa X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Empresa Y</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="saveUser">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>470</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cleanForm">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>470</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="saveFaces">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>390</y>
        <width>361</width>
        <height>53</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 215, 207);</string>
      </property>
      <property name="text">
       <string>Capturar faces do usuário</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Reservas</string>
     </attribute>
     <widget class="QListWidget" name="reserveList">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>981</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>211</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Lista de Reservas</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Cadastrar Reservas</string>
     </attribute>
     <widget class="QWidget" name="reserveForm" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>1011</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="initScheduleLabel">
         <property name="text">
          <string>Início:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="initScheduleLineEdit">
         <property name="minimumSize">
          <size>
           <width>845</width>
           <height>42</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="endScheduleLabel">
         <property name="text">
          <string>Fim:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="endScheduleEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="roomLabel">
         <property name="text">
          <string>Sala:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="roomLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="recorrenteLabel">
         <property name="text">
          <string>Recorrente:?</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="recurrentCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>SIM</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="autorLabel">
         <property name="text">
          <string>CPF Autor:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="autorLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="propositoLabel">
         <property name="text">
          <string>Propósito:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="purposeComboBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Aula</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Evento</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Limpeza</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Individual</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reunião</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCommandLinkButton" name="addPersonsToReserve">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>321</width>
        <height>53</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar Participantes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveReserve">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>480</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cleanReserveForm">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>480</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="reservePersons">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>410</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUserRegister">
   <property name="text">
    <string>Cadastrar Usuário</string>
   </property>
  </action>
  <action name="actionReserveRegister">
   <property name="text">
    <string>Cadastrar Reserva</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
